mods/ccore/functions.lua

```diff
@@ -0,0 +1,5 @@
+-- Public functions for shared use by mods
+function ccore.notify(playername, text, timeout)
+    assert(ccore._notify, "ccore._notify not found")
+    ccore._notify(playername, text, timeout)
```

Comment:
is there a reason why this function is split into 2?
why not integrate `ccore._notify` into `ccore.notify` directly

Reply:
refactored to resolve this.